packages/cli/src/commands/build.js

```diff
@@ -125,6 +126,14 @@ export const handler = async ({
     }
   })
 
+  // Add prebuildTasks
+  listrTasks.unshift({
+    title: 'Running prebuild tasks',
+    task: () => {
+      runPreBuildTasks()
+    },
+  })
+
```

Comment:
Would this be required for building? I guess if someone is building and then running `tsc` to ensure everything is hunky-dory then it's OK?

Reply:
I'm asking cause I want to keep building fast-af, and since it isn't required then we should move it out

Reply:
Hmmm good point, I wanted to make the `preBuildTasks` a generic handler, where we can put any sort of tasks we run before dev/build. Any sort of code generation, etc.

Can take it out if you don't think its worth it.

Reply:
Ok, I've got an idea for something like this - where we have a watcher that runs outside / adjacent to transpilation, I'll write up a ticket

Reply:
Created: https://github.com/redwoodjs/redwood/issues/2219

Reply:
I'll take this out here, I like your idea a lot, dunno why I hadn't thought of it!